mongo/connection_test.go

```diff
@@ -98,7 +98,7 @@ func TestParseConnectionURL(t *testing.T) {
 		t.Fatal("Failed to parse database.")
 	}
 
-	s = "mongodb://user:pass@localhost,1.2.3.4,example.org:1234/another_database?cache=foobar&mode=ro"
+	s = "mongodb://user:pass@localhost:1111,1.2.3.4:2222,example.org:1234/another_database?cache=foobar&mode=ro"
```

Comment:
Could we leave existing tests in place and just append the new ones with the port?